SharedResources/en.lproj/Localizable.strings

```diff
@@ -91,11 +91,14 @@
 "fileProvider.onboarding.button.openCryptomator" = "Open Cryptomator";
 
 "keepUnlocked.alert.title" = "Lock Vault?";
-"keepUnlocked.alert.message" = "Your vault is currently unlocked. To enable this option, your vault has to be locked.";
-"keepUnlocked.main.footer.off" = "If you enable this setting, your vault will be guaranteed to be unlocked for and locked after a certain amount of time.";
-"keepUnlocked.main.footer.on" = "If you disable this setting, your vault may lock prematurely when Cryptomator has been terminated by the Files app. Learn more.";
+"keepUnlocked.alert.message" = "This change requires your vault to be locked in order to take effect after your next unlock.";
+"keepUnlocked.alert.confirm" = "Confirm & Lock Now";
+"keepUnlocked.header" = "Specify how long you want this vault to be unlocked.";
+"keepUnlocked.footer.auto" = "The selected option means that Cryptomator may be terminated in the background by iOS to free up memory, which automatically locks the vault.";
```

Comment:
```suggestion
"keepUnlocked.footer.auto" = "Letting iOS decide means that Cryptomator may be terminated any time to free up memory, which automatically locks the vault.";
```

1. Instead of generic "selected options" we make an explicit reference to _what_ option is selected
2. does "in the background" add any relevant information?